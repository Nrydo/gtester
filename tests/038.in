9212895
668112221
910587891