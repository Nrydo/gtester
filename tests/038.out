9892836590233745